Dockerfile

```diff
@@ -46,9 +46,13 @@ WORKDIR /tmp
 
 # init
 RUN apt update && \
-    apt install -y curl openssl zip gdal-bin wget && \
+    apt install -y curl openssl zip gdal-bin libgdal-grass wget && \
     rm -rf $CATALINA_HOME/webapps/*
 
+RUN curl -q -o /tmp/jni.deb https://nexus.terrestris.de/repository/raw-public/debian/bullseye/libgdal-java_3.2.2-terrestris_amd64.deb
```

Comment:
does it make sense to use the environment variable for llib version in here (like it already done for some other packages above)?

Reply:
Not really, I have to build those debian packages manually for each version, so far only this one exists.